tests/python/frontend/mxnet/test_forward.py

```diff
@@ -1935,6 +1935,29 @@ def verify(data_shape, axis, use_length, length):
     verify((2, 3, 4), 2, True, np.array([[3, 4, 2], [1, 2, 1]]).astype("int32"))
 
 
+@pytest.mark.parametrize(
+    "data_shape, row_sel, col",
+    [
+        ((5, 7), (0, 1, 2, 3, 4,), 2),
+    ],
+)
+@pytest.mark.parametrize("dtype", ["float64", "float32"])
+@tvm.testing.parametrize_targets
+@pytest.mark.parametrize("kind", ["graph", "vm", "debug"])
+def test_forward_npi_advanced_indexing_multiple(data_shape, row_sel, col, dtype, target, ctx, kind):
+    data_np = np.random.uniform(size=data_shape).astype(dtype)
+    data = mx.sym.var("data")
+    ref_res = mx.np.array(data_np)[row_sel, col]
+
+    # TODO need to add the proper symbol operator
+    mx_sym = mx.sym.np.(data.as_np_ndarray()[row_sel, col])
```

Comment:
@junrushao1994, @sxjscience

Need your help to find a proper `mx.sym.np` for the `advanced indexing`, currently the test is not valid yet.

Reply:
You may use this:

```
import mxnet as mx
mx.npx.set_np()
row_sel = mx.sym.var('row_sel').as_np_ndarray()
col = mx.sym.var('col').as_np_ndarray()
data = mx.sym.var('data').as_np_ndarray()
out = data[row_sel, col]
print(out)
```

Reply:
Hi @sxjscience

Thank you for the suggestion; however, I still not clear how to create `mx_sym` so that I can use it for `relay.frontend.from_mxnet`.

Any suggestion would appreciated.
Thank you.

Reply:
Is it possible to do the following?
```python
row_sel_sym = mx.sym.var('row_sel').as_np_ndarray()
col_sym = mx.sym.var('col').as_np_ndarray()
data_sym = mx.sym.var('data').as_np_ndarray()
mx_sym = data_sym[row_sel_sym, col_sym]

```

Then, you may pass `mx_sym` to the `relay.frontend.from_mxnet`.

Reply:
Hi @sxjscience 

Thank you for the suggestion. I think I had tried that as well.
So with the following, I got this exception `IndexError: Only integer, slice, or tuple of these types are supported! Received key=(<_Symbol row_sel>, <_Symbol col>)` from `mxnet/symbol/numpy/_symbol.py:135:` ([link](https://github.com/apache/incubator-mxnet/blob/124d8417984ed9205f972eb1c2cadbf028b94eb3/python/mxnet/symbol/numpy/_symbol.py#L149))

I will dig more, but if you have any suggestion, please let me know.

```
def test_forward_npi_advanced_indexing_multiple(data_shape, row_sel, col, dtype, target, ctx, kind):
    data_np = np.random.uniform(size=data_shape).astype(dtype)
    ref_res = mx.np.array(data_np)[row_sel, col]

    row_sel_sym = mx.sym.var("row_sel").as_np_ndarray()
    data_sym = mx.sym.var("data").as_np_ndarray()
    col_sym = mx.sym.var("col").as_np_ndarray()
    mx_sym = data_sym[row_sel_sym, col_sym]

    mod, _ = relay.frontend.from_mxnet(
        mx_sym, shape={"data": data_shape, "row_sel": row_sel, "col": col}, dtype=dtype
    )
    intrp = relay.create_executor(kind, mod=mod, ctx=ctx, target=target)
    op_res = intrp.evaluate()(data_np)
    tvm.testing.assert_allclose(op_res.asnumpy(), ref_res.asnumpy(), rtol=1e-5)
    tvm.testing.assert_allclose(ref_res.asnumpy(), ref_res.asnumpy(), rtol=1e-5)


```

Reply:
Also, I think you will need to install the nightly version of MXNet 2.0. You may follow the guide in https://github.com/dmlc/gluon-nlp.

```bash
# Install the version with CUDA 10.1
python3 -m pip install -U --pre "mxnet-cu101>=2.0.0b20201206" -f https://dist.mxnet.io/python

# Install the version with CUDA 10.2
python3 -m pip install -U --pre "mxnet-cu102>=2.0.0b20201206" -f https://dist.mxnet.io/python

# Install the version with CUDA 11
python3 -m pip install -U --pre "mxnet-cu110>=2.0.0b20201206" -f https://dist.mxnet.io/python

# Install the cpu-only version
python3 -m pip install -U --pre "mxnet>=2.0.0b20201206" -f https://dist.mxnet.io/python

```

Reply:
Thank you, I have updated the MXNet; however, something had happened to my workspace.
I kept getting this, let me try few more things, and get back on the test case.
```
..
Check failed: it != type_key2index_.end() == false: Cannot find type auto_scheduler.PythonBasedMeasureCallback. Did you forget to register the node by TVM_REGISTER_NODE_TYPE ?
```